packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7962,11 +7966,7 @@ export default class Meeting extends StatelessWebexPlugin {
    *
    * @returns {object} containing timeLeft, expiryTime, currentTime
    */
-  public getPermissionTokenExpiryInfo(): {
-    timeLeft?: number;
-    expiryTime?: number;
-    currentTime: number;
-  } {
+  public getPermissionTokenExpiryInfo() {
```

Comment:
wouldn't it be simpler to just change code in line 7971 to `return {};`?

Reply:
We decided against that, if there is no payload then that means there is nothing and returning empty object is not ideal